### PR TITLE
add release scripts

### PR DESCRIPTION
**Pull request by kunmingg**
Add release script for releasing katib images.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/675)

### CODE REVIEW

File: submit_release_job.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+JOB_NAME="katib-release"
+JOB_TYPE=katib-release
+BUILD_NUMBER=$(uuidgen)
+BUILD_NUMBER=${BUILD_NUMBER:0:4}
+REPO_OWNER=kubeflow
+REPO_NAME=katib
+ENV=test
```
Comment (hougangliu) [inline submit_release_job.sh]: I think we'd better use `release` ENV instead

**Comment by hougangliu** [comment]
@kunmingg please also rebase your PR to make kubeflow-katib-presubmit passed

File: submit_release_job.sh (hunk 1 above)
Comment (kunmingg) [reply submit_release_job.sh]: Ideally we should use `release` ENV, but it's been out of date for a while...
We need to rebuild release infra, and before that `test` ENV also work for release workflows.

Comment (hougangliu) [reply submit_release_job.sh]: sgtm, so please rebase your PR to make kubeflow-katib-presubmit passed (a fix to the error has been merged into master branch)

**Pull request closed without merging**